### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by Risengan** [mrkai77/Loop#456] [Q&A]
I'd be happy to help with the French translation! 🇫🇷

**Comment by mrkai77** [comment]
Merci beaucoup! I will be sending over a translation file soon :)
In the meantime, please make sure to install Xcode!

**Reply by mrkai77** [reply]
Hey @Risengan! If you have Discord, please join our server [here](https://discord.gg/7bGm4G83FP)! I will be sending localization files there within the next few days, since it will be easier to help with different contexts there :)
